debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat

```diff
@@ -34,6 +34,11 @@
 
 CLimits.USHRT_MAX = 0xFFFF
 
+/* The below matches the current flags in the internal builds.
+ * They should be updated if the OJ9 MHs are disabled and
+ * OJDK MHs are enabled in the internal builds.
```

Comment:
Could you please spell method handles, OpenJ9 and OpenJDK in full?

Reply:
The comment also applies to `J9VM_OPT_OPENJDK_METHODHANDLE`; when internal builds switch, both should be updated.

Reply:
Updated.